src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadoc1.java

```diff
@@ -13,7 +13,7 @@ public class InputSummaryJavadoc1 {
     // violation 2 lines below 'Forbidden summary fragment'
     /**
      * {@summary Returns the customer ID.
-     *  This method returns. }
+     *  This method returns.}
```

Comment:
We almost never change inputs, the more they weird the better.
Please change list of violations.

Reply:
Done